sdk/servicebus/service-bus/package.json

```diff
@@ -102,11 +116,12 @@
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
     "preinstall": "node ../../../common/scripts/rush-welcome.js",
+    "pretest": "npm run build-test",
     "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
     "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
-    "unit-test:browser": "echo skipped",
-    "unit-test:node": "echo skipped",
+    "unit-test:node": "npm run build-test && mocha -t 120000 test-dist/index.js",
```

Comment:
Don't run `build:test` within the unit-test or integration-test commands, the build system will do that automatically.

Also make sure these are sorted alphabetically.